### DISCUSSION TITLE
Running Z3 C++ in macOS Ventura

### DISCUSSION THREAD

**Discussion by anirjoshi** [Z3Prover/z3#6903] [Q&A]
Hello,
 I am having issues running Z3 c++ program on macOS. Here is the error the I get when I use `g++ example.cpp`:
```
Undefined symbols for architecture arm64:
  "_Z3_add_const_interp", referenced from:
      __ZN2z35model16add_const_interpERNS_9func_declERNS_4exprE in cc9LVDmr.o
  "_Z3_add_rec_def", referenced from:...
"_Z3_translate", referenced from:
      __Z21two_contexts_example1v in cc9LVDmr.o
ld: symbol(s) not found for architecture arm64
collect2: error: ld returned 1 exit status
```

When I run the command `g++ example.cpp -lz3` I get the following error:
```
ld: library not found for -lz3
collect2: error: ld returned 1 exit status
```

I am not sure how it is able to find the ".h" file but not the library paths. Some help would be great. Thanks!

**Comment by wintersteiger** [comment]
Perhaps missing a `-L <directory containing libz3.dylib>`?
